doc/admin-guide/files/records.yaml.en.rst

```diff
@@ -4579,6 +4579,16 @@ HTTP/2 Configuration
    Clients exceeding this limit will be immediately disconnected with an error
    code of ENHANCE_YOUR_CALM.
 
+.. ts:cv:: CONFIG proxy.config.http2.max_empty_frames_per_minute INT 0
+   :reloadable:
+
+   Specifies how many empty frames |TS| receives for a minute at maximum.
+   What "empty frames" means here is DATA frame that does not carry payload
+   nor END_STREAM flag, and CONTINUATION frame that does not carry payload
+   nor END_HEADERS flag.
+   Clients exceeded this limit will be immediately disconnected with an error
+   code of ENHANCE_YOUR_CALM.
+
```

Comment:
It would be good to document that a negative value means unlimited for these relevant configurations. A default statement would also be helpful. Maybe something like:

```
Any negative value configures no limit to the number of empty frames received. ``0`` is 
the default configuration, meaning that no empty frames are allowed.
```